src/Sylius/Behat/Context/Setup/ShopSecurityContext.php

```diff
@@ -80,5 +83,13 @@ public function iAmLoggedInCustomerByUsingRememberMeOption(): void
         $this->userRepository->add($user);
 
         $this->securityService->logInWithRememberMe($user);
+
+        $this->storeUserToken($user);
```

Comment:
Don't think this one is necessary;
Can you even use remember me through the API?